prerelease_script.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+
+def compilemessages(data):
+    try:
+        if data['workingdir'] != data['reporoot']:
+            raise EnvironmentError('Working dir is not the root one')
+        os.chdir("mds")
+        os.system("../manage.py compilemessages")
```

Comment:
That is a bit dirty. Why do you need to change repertory ? If it's to find the translations, try setting https://docs.djangoproject.com/fr/2.1/ref/settings/#locale-paths to the correct path

Reply:
Also try with this cleaner version:
```python
import subprocess

subprocess.call(['django-admin', 'compilemessages'])
```

Reply:
I need to change dir because it needs to be done ine the mds folder of the project. If it is done outside, then the .mo files won't be generated (don't know why).

Reply:
Did you try to set `LOCALE_PATHS` ?

Reply:
My bad, it seems like doing compilemessages works (kinda weir, I remember having some kind of trouble about it).
Just gonna change os.system to subprocess then